plugins/git/git.plugin.zsh

```diff
@@ -79,6 +79,7 @@ alias gd='git diff'
 alias gdca='git diff --cached'
 alias gdct='git describe --tags `git rev-list --tags --max-count=1`'
 alias gdt='git diff-tree --no-commit-id --name-only -r'
+alias gdtl='git difftool'
```

Comment:
Currently there are two identical aliases in the cheatsheet:

https://github.com/robbyrussell/oh-my-zsh/wiki/Cheatsheet#git

gdt => git diff-tree --no-commit-id --name-only -r
gdt => git difftool